src/js/SoftButtons.js

```diff
@@ -14,31 +14,43 @@ export default class SoftButtons extends React.Component {
             items = softButtons.map((softButton, index) => {
                 return (<div className="soft-button-tile-large th-f-color t-small t-light th-bg-color th-soft-buttons" 
                             key={softButton.softButtonID}
-                            onClick={() => this.props.onButtonPress(this.props.appID, softButton.softButtonID, "CUSTOM_BUTTON")}>
+                            /*onClick={() => this.props.onButtonPress(this.props.appID, softButton.softButtonID, "CUSTOM_BUTTON")}*/
+                            onMouseDown={() => this.props.onButtonDown(this.props.appID, softButton.softButtonID, "CUSTOM_BUTTON")}
+                            onMouseUp={() => this.props.onButtonUp(this.props.appID, softButton.softButtonID, "CUSTOM_BUTTON")}                           
+                        >
```

Comment:
Is it possible to return this in a function? It seems like this code is repeated in quite a few places

Reply:
Also, remove commented `onClick` function